pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -305,25 +306,30 @@ def store_structure(self):
 
     def animate_structure(
         self,
-        spacefill=True,
-        show_cell=True,
-        stride=1,
-        center_of_mass=False,
-        particle_size=0.5,
-        camera="orthographic",
+        spacefill: bool = True,
+        show_cell: bool = True,
+        stride: int = 1,
+        center_of_mass: bool = False,
+        particle_size: float = 0.5,
+        camera: str = "orthographic",
+        atom_indices: Union[list, np.ndarray] = None,
+        snapshot_indices: Union[list, np.ndarray] = None
     ):
         """
         Animates the job if a trajectory is present
 
         Args:
-            spacefill (bool):
-            show_cell (bool):
+            spacefill (bool): If True, then atoms are visualized in spacefill stype
+            show_cell (bool): True if the unitcell is to be shown
             stride (int): show animation every stride [::stride]
                           use value >1 to make animation faster
                            default=1
-            center_of_mass (bool):
+            particle_size (float): Radius of the spacefills
```

Comment:
This is slightly outside of this PR, but `particle_size` in `plot3d()` is not  the radius, but the multiplication factor of the base radius, which is given by the atomic number. It might be helpful to add it at some point (or maybe in this PR already), because I find it difficult to understand in `animate_structure` when small atoms like H or C are shown in the same size as Ni or Fe.

Reply:
Thanks. I've changed the docstrings